src/components/pages/MentorMenteeMatching/MentorMenteeMatching.js

```diff
@@ -29,55 +78,32 @@ const MentorMenteeMatching = () => {
       ),
     },
     { title: 'Stack', dataIndex: 'stack', key: 'stack' },
+    {
+      title: '',
+      dataIndex: 'name',
+      key: 'assignButton',
+      render: (text, record) => (
+        <Button size="large" type="primary" onClick={() => showModal(record)}>
+          Assign Mentor
+        </Button>
+      ),
+    },
   ];
 
-  const data = [];
-
   // adds commas in-between the programming languages
-  const splitTechStack = obj => {
-    let string = '';
-    for (let i = 0; i < obj.tech_stack.length; i++) {
-      if (i === obj.tech_stack.length - 1) {
-        string += `${obj.tech_stack[i]}`;
-      } else string += `${obj.tech_stack[i]}, `;
-    }
-    return string;
+  const splitTechStack = techStackArray => {
```

Comment:
great clean code, thats saving 8 lines of code into 2 and its much much cleaner, thank you for that.